viewer/src/components/Charts.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="gaia-viewer-modal">
+    <div id="player-selection">
+      <span
+        v-for="index in players"
+        :key="index"
+        @click="
+          $data.selected = index;
+          selectPlayer(index);
+        "
+      >
+        <svg :x="(index > 3 ? -1 : index) * 2" y="0" width="80" height="80" viewBox="-1.5 -1.5 4 4">
+          <PlayerCircle
+            :player="index > 3 ? null : gameData.player(index)"
+            :index="index"
+            :class="`chart-player ${index === $data.selected ? 'selected' : ''}`"
+            translate=""
+            chart="true"
+          />
+        </svg>
+      </span>
+    </div>
+
+    <canvas id="graphs" />
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+import { makeChart, victoryPointDetails, victoryPointSummary } from "../logic/charts";
+import PlayerCircle from "./PlayerCircle.vue";
+import Engine, { PlayerEnum } from "@gaia-project/engine";
+import Chart from "chart.js";
+
+
+@Component({
+  components: { PlayerCircle },
+})
+export default class Charts extends Vue {
+
+  data() {
+    return {
+      selected: PlayerEnum.All,
+    };
+  }
```

Comment:
```suggestion
  selected = PlayerEnum.All
```

If you'd really wanted to use data(), it would have been better in the `@Component` part, but there's no reason to:

```ts
@Component({
  data() {
    // values returned are then usable directly with `this`
    return {...}
   }
}
```

With the Typescript class syntax for vue, you can just declare member variables with an initial value instead.

Reply:
I thought this would somehow not work - but it was probably just the auto-complete that didn't work..